### PR TITLE
Limit Highlight.js language detection

### PR DESCRIPTION
**Pull request by Twaha-Rahman**
This PR addresses #40

### CODE REVIEW

File: src/utilities/process-reply.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ module.exports = async function processReply({
         if (supportedLanguage) {
           const formattedCode = formatter(content, supportedLanguage);
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[0], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```
Comment (bradtaniguchi) [inline src/utilities/process-reply.js:41]: If we are just using `supportedLanguage` now, what about the `languageGuesses` param? Does the bot no longer ever give multiple language suggestions?

File: src/utilities/process-reply.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ module.exports = async function processReply({
           const formattedCode = formatter(content, supportedLanguage);
 
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[1], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```
Comment (bradtaniguchi) [inline src/utilities/process-reply.js:56]: This no longer seems possible, if we are in the `else` case but the `if` case is if there is a `supportedLanguage` in the first place?

File: src/utilities/process-reply.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ module.exports = async function processReply({
         if (supportedLanguage) {
           const formattedCode = formatter(content, supportedLanguage);
           ref.channel.send(
-            formatMessageWithCodeblock(languageGuesses[2], formattedCode)
+            formatMessageWithCodeblock(supportedLanguage, formattedCode)
```
Comment (bradtaniguchi) [inline src/utilities/process-reply.js]: Isn't this also impossible?

**Review by bradtaniguchi — CHANGES_REQUESTED** [review]

File: src/utilities/process-reply.js (hunk 3 above)
Comment (Twaha-Rahman) [reply src/utilities/process-reply.js]: This was the third option when `language-classifier` was used....yes this code is unreachable... I'll look into it in more details after a few hours...

**Comment by Twaha-Rahman** [comment]
@bradtaniguchi The bot gives multiple suggestions just like before..... I have only changed the `languageGuesses[0]` and `languageGuesses[1]` with `supportedLanguage` (If the language can be formatted with `Prettier`) . This change is made to change some detected languages name when we return the message to Discord, because even though `Prettier` supports the lang, Discord doesn't support it. For example- `SCSS` gets's detected and formatted with `Prettier` but Discord doesn't support syntax highlighting for it..... so we change `SCSS` to `CSS` when we format the message for Discord.

In this line `const supportedLanguage = isSuppotedByPrettier(languageGuesses[0]);` the `isSuppotedByPrettier` function checks if the lang is supported by `Prettier`.... if it isn't, the function returns false... that's why we check if the variable `supportedLanguage ` is false...

File: src/utilities/process-reply.js (hunk 3 above)
Comment (Twaha-Rahman) [reply src/utilities/process-reply.js]: I've removed the third `case` in the `switch` statement as it is unreachable after removing `language-classifier`.

File: src/utilities/process-reply.js (hunk 1 above)
Comment (Twaha-Rahman) [reply src/utilities/process-reply.js:41]: I've explained this details in the message below...

**Review by bradtaniguchi — APPROVED** [review]
Just tested this in my bot room again and it looked to work with a I posted earlier and it worked. I also accidentally tested this on an already formatted message provided from the bot, and the formatted code ended up really funky, but I don't think that is that big of a deal.

LGTM :+1:

**Pull request merged**